examples/others/imgfont/lv_example_imgfont_1.py

```diff
@@ -31,6 +31,7 @@ def get_imgfont_path(font, img_src, length, unicode, unicode_next,user_data) :
 imgfont = lv.imgfont_create(80, get_imgfont_path, None)
 if imgfont == None:
     print("imgfont init error")
+    return
```

Comment:
```suggestion
```

Reply:
If imgfont creation fails, execution of the following code should be stopped.

Reply:
Maybe we want `sys.exit()`? 

@amirgon @uraich what is the best way to abort on error in MicroPython?

Reply:
> @amirgon @uraich what is the best way to abort on error in MicroPython?

`sys.exit()` should work.  
Or you can simply condition the few lines on the following code.

Reply:
If we use `return`, it may keep the same behaviour with `C` example version.

Reply:
> If we use return, it may keep the same behaviour with C example version.

But this part is not inside of a function, so `return` is invalid.

Reply:
Ooops, yes.  I am seeing things 🥲

And just `exit()` is ok.